troubleshooting.adoc

```diff
@@ -279,3 +279,12 @@ image::./images/invalid_bind_credentials.png["Error - Invalid bind credentials"]
 ** Check that the correct certificate was provided in the secret that is referenced in the `ca` field  for the ldap identity provider in the AuthRealm CR.
 +
 image::./images/ldap_ca_error.png["Error - Incorrect certificate for LDAP server"]
+
+[#Import groups troubleshoot]
+=== troubleshoot import groups
+
+==== Groups are not created.
+
+* Check the dex-server logs if the groups are displaied, if not you probably have an issue with the relationship between the user and the group in your IDP configuration. For example for LDAP, the user is not a member of the group.
+
+* Check the authentication-operator log for errors.
```

Comment:
Can we add a hyperlink here to the section above where looking up authentication-operator logs is explained?

Reply:
Done